pkg/pipelines/create_cluster.go

```diff
@@ -83,6 +83,7 @@ func NewCreateClusterPipeline(runtime *common.KubeRuntime) error {
 		&kubernetes.StatusModule{},
 		&kubernetes.JoinNodesModule{},
 		&loadbalancer.HaproxyModule{Skip: !runtime.Cluster.ControlPlaneEndpoint.IsInternalLBEnabled()},
+		&loadbalancer.KubevipModule{Skip: !runtime.Cluster.ControlPlaneEndpoint.IsInternalLBEnabledVip() && runtime.Cluster.ControlPlaneEndpoint.Address != ""},
```

Comment:
IMO, it's better to add a new module that may be named`LoadBalancerModule`. And we can add control statements in there to allow kk to install different load balancers. The pipeline only needs to add the `LoadBalancerModule` just like the `InstallContainerModule`.